packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -464,9 +484,6 @@ export const InlineEditingMixin = (superClass) =>
         const nextCell = Array.from(nextRow.children).find((cell) => cell._column === nextCol);
         e.preventDefault();
```

Comment:
`preventDefault` is still called unconditionally here. Intended? Is there some issue with the test setup if that isn't caught?

Reply:
I think it's not needed here. Removed. 

Not sure if this would actually work as we first wait for native `focusout` so at this point trying to `preventDefault()` shouldn't be actually needed.